### PR TITLE
LPD-28975 Workflow tab shows unexpected error for asset added in publication and subsequently enabling workflow

### PR DESCRIPTION
**Pull request by gislayne-vitorino**
Update for: [LPD-28975](https://liferay.atlassian.net/browse/LPD-28975)

Now the workflow tab does not appear when viewing a change that was added before workflow were enabled for its entity.

The issue was related to the code only checking if workflow was enabled and not if there were tasks related to the change. 

image

### CODE REVIEW

**Comment by gislayne-vitorino** [comment]
ci:test:publications

**Review by ctang3 — APPROVED** [review]
Confirmed functionality on my local and code changes look good to me. Nice work!

**Comment by ctang3** [comment]
ci:test:relevant

File: modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/ViewChangesDisplayContext.java [hunk 1]
```diff
@@ -120,7 +125,9 @@ public ViewChangesDisplayContext(
 		GroupLocalService groupLocalService, Language language, Portal portal,
 		PublicationsDisplayContext publicationsDisplayContext,
 		PublishScheduler publishScheduler, RenderRequest renderRequest,
-		RenderResponse renderResponse, UserLocalService userLocalService) {
+		RenderResponse renderResponse, UserLocalService userLocalService,
+		WorkflowTaskManager workflowTaskManager,
+		WorkflowInstanceLinkLocalService workflowInstanceLinkLocalService) {
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/ViewChangesDisplayContext.java:130]: Sort

**Review by pei-jung — COMMENTED** [review]
Changes look good except one minor SF change request. 👍 

@gislayne-vitorino please add a test for this.

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Review by pei-jung — APPROVED** [review]

**Comment by pei-jung** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
ci:test:publications

**Comment by gislayne-vitorino** [comment]
ci:report:29284353

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:publications

**Comment by gislayne-vitorino** [comment]
ci:report:31133025

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:reevaluate:1297401_10008

**Comment by pei-jung** [comment]
ci:test:publications

**Comment by pei-jung** [comment]
ci:stop

**Comment by pei-jung** [comment]
@gislayne-vitorino there is a merge conflict.

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:publications

**Comment by pei-jung** [comment]
ci:forward

**Pull request closed without merging**